test/cmd/node-management.sh

```diff
@@ -120,16 +126,16 @@ __EOF__
   kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.unschedulable}}" '<no value>'
   # Pre-condition: test-pod-1 and test-pod-2 exist
   kube::test::get_object_assert "pods" "{{range .items}}{{.metadata.name}},{{end}}" 'test-pod-1,test-pod-2,'
-  # dry-run command
-  kubectl drain "127.0.0.1" --pod-selector 'e in (f)' --dry-run=client
-  kubectl drain "127.0.0.1" --pod-selector 'e in (f)' --dry-run=server
+  # dry-run command - need to use force because pods are unmanaged
+  kubectl drain "127.0.0.1" --pod-selector 'e in (f)' --dry-run=client --force
+  # TODO: reinstate this when server-side dry run is fixed. See https://github.com/kubernetes/kubernetes/pull/100206
+  # kubectl drain "127.0.0.1" --pod-selector 'e in (f)' --dry-run=server --force
   kube::test::get_object_assert "pods" "{{range .items}}{{.metadata.name}},{{end}}" 'test-pod-1,test-pod-2,'
-  # command
-  kubectl drain "127.0.0.1" --pod-selector 'e in (f)'
+  # command - need to use force because pods are unmanaged
+  # TODO: reinstate this when it works on CI. See https://github.com/kubernetes/kubernetes/pull/100206
```

Comment:
As mentioned in https://github.com/kubernetes/kubernetes/pull/100206, when I fix this suite so that the test pods are actually on the target node, the drain command seems to hang forever in CI specifically, and I haven't figured out why. 

We can block this PR on the other one if you'd like, but please note that this assertion isn't proving all that much right now, with no pods on the node.